### ISSUE TITLE
Adjust todo list progress?

### ISSUE THREAD

**Issue by kazerniel** [MadeBaruna/paimon-moe#214]
Say, I made partial progress on my todo list, but then I change my mind and remove a character/weapon and add others, or even just use up a required material for cooking.

I seem to have no way to adjust the amount I have of a required item without removing all targets that require it.

It would also help if the todo cards listed how many of the item was originally required, eg. "0/6 x Item", instead of just "0 x Item".

**Comment by Robik81** [comment]
Yes! Your suggested changes are really desirable. Right now I often find best to only add items to todo list, but never substract amount of obtained material to keep total amount needed for character / weapon. And mainly use it for an overview what I can farm today from materials I need.

**Comment by kazerniel** [comment]
@Robik81: Since my post I found [seelie.me](https://seelie.me/) and have been using it for a year now. It's perfect for this purpose :)

**Comment by Robik81** [comment]
@kazerniel Thanks! I'll check it out.
